### PR TITLE
Change proptype of errortext since it is now always an array

### PR DESCRIPTION
**Pull request by summer-cook**

### CODE REVIEW

**Review by labradford — APPROVED** [review]

File: src/components/Error/Error.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const Error = ({ errors, showBackButton, canDismissAlert, router }) => {
 Error.propTypes = {
   canDismissAlert: PropTypes.bool,
   errors: PropTypes.shape({
-    errorText: PropTypes.string.isRequired,
+    errorText: PropTypes.array.isRequired,
```
Comment (alishaevn) [inline src/components/Error/Error.jsx:39]: can you use `arrayOf` and define the array, like we do with objects? there are examples of it in the code if you need it.

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: src/components/Error/Error.jsx (hunk 1 above)
Comment (summer-cook) [reply src/components/Error/Error.jsx]: done

File: src/components/Error/Error.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const Error = ({ errors, showBackButton, canDismissAlert, router }) => {
 Error.propTypes = {
   canDismissAlert: PropTypes.bool,
   errors: PropTypes.shape({
-    errorText: PropTypes.string.isRequired,
+    errorText: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```
Comment (alishaevn) [inline src/components/Error/Error.jsx:39]: re: [my prior request](https://github.com/scientist-softserv/webstore-component-library/pull/131#discussion_r1084568484)

isn't errorText an [array on strings](https://github.com/scientist-softserv/webstore/blob/main/utils/api/configurations.js#L72)?

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: src/components/Error/Error.jsx (hunk 2 above)
Comment (summer-cook) [reply src/components/Error/Error.jsx]: yes definitely, i was thinking of errors! my bad just changed it

File: src/components/Error/Error.jsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ const Error = ({ errors, showBackButton, canDismissAlert, router }) => {
 Error.propTypes = {
   canDismissAlert: PropTypes.bool,
   errors: PropTypes.shape({
-    errorText: PropTypes.string.isRequired,
+    errorText: PropTypes.arrayOf(PropTypes.shape('')).isRequired,
```
Comment (alishaevn) [inline src/components/Error/Error.jsx:39]:
```suggestion
    errorText: PropTypes.arrayOf(PropTypes.string).isRequired,
```

**Review by alishaevn — CHANGES_REQUESTED** [review]

**Review by alishaevn — APPROVED** [review]

**Pull request merged**